test/Microsoft.AspNetCore.Razor.Test/CodeGenerators/CSharpRazorCodeGeneratorTest.cs

```diff
@@ -4,13 +4,13 @@
 using System;
 using System.Collections.Generic;
 using Microsoft.AspNetCore.Razor.Chunks.Generators;
-using Microsoft.AspNetCore.Razor.CodeGenerators;
 using Microsoft.AspNetCore.Razor.Parser.SyntaxTree;
+using Microsoft.AspNetCore.Razor.Test.CodeGenerators;
```

Comment:
I moved the classes that test a specific implementation. However, there's infrastructure bits that are only test related. Do we typically move infrastructure bits into their corresponding src namespaces.